src/servidor/ServidorService.ts

```
@@ -0,0 +1,34 @@
+import { BaseService } from '@/utils/BaseService.ts'
+import { Servidor } from '@/commons/interfaces.ts'
+
+class ServidorService extends BaseService {
+  constructor() {
+    super('/usuarios/servidor')
+  }
+
+  async save(servidor: Servidor): Promise<void> {
+    const response = await this.post<{ token?: string }, Servidor>(servidor)
+    if (response && response.token) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)